docs/deployment/nio/release.md

```diff
@@ -0,0 +1,39 @@
+# Release Cycle
+
+Releasing a new or updated nio instance is similar to releasing a new version on github.
+
+## Doing a Release
+
+To do the first release, first click the Release button in the Instance toolbar:
+
+<img class="left" src="/img/deploy/release/toolbar.png" />
+
+### Creating a new Instance Configuration
+
+This will show the option to create a a new Instance Configuration. Give your Instance Configuration a name and an initial version. We recommend `v1.0.0` which follows the [semver](https://semver.org) scheme.
+
+<img class="left" src="/img/deploy/release/modal.png" height="350" />
+
+Click `OK` to release your first Instance Configuration!
+
+If you would now like to deploy, you should move on to:
+
+* [Direct Deployment](/deployment/nio/direct.md)
+* [Release Deployment](/deployment/nio/release.md)
```

Comment:
Did you mean to link to the page the user is on?

Reply:
Definitely not. Great catch!